changes_summary.md

```
@@ -0,0 +1,365 @@
+
+# With version tags
+docker build \
+  --build-arg GIT_TAG=$(git describe --tags) \
+  --build-arg GIT_COMMIT=$(git rev-parse HEAD) \
+  --target runtime-ceph \
+  -t prysm-ceph:$(git describe --tags) .
+```
+
+### Build Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)